action.yaml

```diff
@@ -315,6 +315,14 @@ runs:
 
         DBT_DEPLOY=false
 
+        # case when the triggered event is a manual workflow dispatch, we would need to deploy the dbt project because we cannot determine that it does not need to be deployed
+        GITHUB_EVENT_BEFORE=${{ github.event.before }}
+        GITHUB_EVENT_AFTER=${{ github.event.after }}
+        if [[ -z $GITHUB_EVENT_BEFORE && -z $GITHUB_EVENT_AFTER ]]; then
```

Comment:
nit: could we club both the `if` clause (this and the one below) into a single clause to avoid code repetition :)

Reply:
Addressed